File: java/src/main/java/br/com/acqio/dto/StatusResponse.java
package br.com.acqio.dto;

import lombok.Getter;
import lombok.Setter;
import lombok.ToString;

import java.util.List;

@Getter
@Setter
@ToString
public class StatusResponse {
    private Long id;
    private Integer status;
    private List<String> actions;
    private String message;
    private List<InstallmentInfo> installmentInfo;
    private CardInfo cardInfo;
}

File: csharp/csharp/dto/StatusResponse.cs
using System.Collections.Generic;

namespace csharp.dto
{
    class StatusResponse
    {
        public long id { get; set; }
        public int status { get; set; }
        public List<string> actions { get; set; }
        public string message { get; set; }
        public List<InstallmentInfo> installmentInfo { get; set; }
        public CardInfo cardInfo { get; set; }

    }
}